doc/changes/changes_3.0.0.md

```diff
@@ -0,0 +1,13 @@
+# Virtual Schema Common Document 3.0.0, released 2020-XX-XX
+
+Code name: Added optional SOURCE_REFERENCE column
+
+## Features / Enhancements
+
+* #36: Added optional SOURCE_REFERENCE column
+  * see the [user-guide](../user_guide/edml_user_guide.md#source-reference-column)
+  * Caused new EDML version 1.3.0
```

Comment:
I am not sure sub points necessary here. Maybe you can add short description of column and notice to check the user guide for more info in summary section above.

Reply:
The version of new EDML should be `1.2.0`, right? I see it is 1.2.0 in deployment and also in code.